motoman_ms210_moveit_config/config/sensors_3d.yaml

```
@@ -0,0 +1,3 @@
+# The name of this file shouldn't be changed, or else the Setup Assistant won't detect it
+sensors:
+  - {}
```

This will cause warnings/errors when this file gets loaded.

```suggestion
sensors: []
```